lego/apps/events/models.py

```diff
@@ -871,20 +909,20 @@ def should_notify(self, time=None):
             )
         return False
 
-    def set_payment_success(self):
+    def set_payment_success(self) -> Registration:
         self.payment_status = constants.PAYMENT_MANUAL
         self.save()
         return self
 
-    def set_presence(self, presence):
+    def set_presence(self, presence: str) -> None:
```

Comment:
`presence` must be in `constants.PRESENCE_CHOICES`, but I couldn't get to type it any better than this. Would love some input.

Reply:
I would make `constants.PRESENCE_CHOICES` an `Enum` and type everything accordingly. That would give type safety and make everything alot cleaner ;)